### PR TITLE
New set of Linux quizzes

### PR DESCRIPTION
**Pull request by mihaiberq**

### CODE REVIEW

**Comment by enki-hq** [comment]
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/linux/jobs-and-processes-data-manipulation/crontab/common-issue-with-scripts-in-cron.md?ref=70f12883a65578b8d6667c4dccc9dbe70fe84cae
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/linux/networking-and-security/ssh-remote/proxifying-ssh-connections.md?ref=70f12883a65578b8d6667c4dccc9dbe70fe84cae
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/linux/user-and-file-management/bash-environment/understanding-the-path-environment-variable.md?ref=70f12883a65578b8d6667c4dccc9dbe70fe84cae
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/linux/user-and-file-management/conditions/the-test-built-in.md?ref=70f12883a65578b8d6667c4dccc9dbe70fe84cae
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/linux/user-and-file-management/file-management/using-which-to-find-executables.md?ref=70f12883a65578b8d6667c4dccc9dbe70fe84cae
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/linux/user-and-file-management/terminal-power-user-tips/useful-aliasing-in-bash.md?ref=70f12883a65578b8d6667c4dccc9dbe70fe84cae
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/linux/user-and-file-management/user-management/changing-user-passwords-with-passwd.md?ref=70f12883a65578b8d6667c4dccc9dbe70fe84cae
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/linux/user-and-file-management/user-permissions/changing-file-permissions-with-chmod.md?ref=70f12883a65578b8d6667c4dccc9dbe70fe84cae

File: linux/user-and-file-management/user-management/changing-user-passwords-with-passwd.md [hunk 1]
```diff
@@ -41,17 +41,30 @@ Change `enkiUser` ‘s password:
...
-*No
-*Yes
\ No newline at end of file
+* No
+* Yes
+
+---
+## Quiz
+
+headline: which of the following practices should be avoided?
```
Comment (lizTheDeveloper) [inline linux/user-and-file-management/user-management/changing-user-passwords-with-passwd.md]: Can you identify a security vulnerability?

File: linux/user-and-file-management/file-management/using-which-to-find-executables.md [hunk 2]
```diff
@@ -49,11 +49,24 @@ There are two major use cases for `which`:
...
+* which
+* try
+* ps
+* where
+* man
+
+---
+## Quiz
+
+headline: finding executables location
```
Comment (lizTheDeveloper) [inline linux/user-and-file-management/file-management/using-which-to-find-executables.md]: Do you know how to find the location of an executable?

File: linux/user-and-file-management/bash-environment/understanding-the-path-environment-variable.md [hunk 3]
```diff
@@ -65,19 +65,39 @@ In what order will the shell visit the folders in `PATH` to find a valid executa
...
-*in the list of folders contained in the EXE variable.
+* in the list of folders contained in the PATH variable.
+* anywhere on the system.
+* in the local folder.
+* in the list of folders contained in the EXE variable.
+
+---
+## Quiz
+
+headline: how does $PATH work?
```
Comment (lizTheDeveloper) [inline linux/user-and-file-management/bash-environment/understanding-the-path-environment-variable.md]: Do you know how the $PATH variable is evaluated?

File: linux/user-and-file-management/terminal-power-user-tips/useful-aliasing-in-bash.md [hunk 4]
```diff
@@ -75,18 +75,41 @@ Complete the following snippet:
...
\ No newline at end of file
+* alias
+* short
+* rm
+* ctrl+k
+
+---
+## Quiz
+
+headline: what is going on with this command?
```
Comment (lizTheDeveloper) [inline linux/user-and-file-management/terminal-power-user-tips/useful-aliasing-in-bash.md]: Can you figure out what a command does given only it's output?

File: linux/user-and-file-management/user-permissions/changing-file-permissions-with-chmod.md [hunk 5]
```diff
@@ -70,22 +70,42 @@ As we add and remove permissions, you can see the changes reflected in the permi
...
\ No newline at end of file
+* Gives execute permissions to group and others
+* Removes execute permissions from group and others
+* Gives write and read permissions to user
+* Removes write and read permission from group and others
+
+---
+## Quiz
+
+headline: how would you run the following script?
```
Comment (lizTheDeveloper) [inline linux/user-and-file-management/user-permissions/changing-file-permissions-with-chmod.md]: How do you run a script with elevated permissions?

**Pull request merged**